### ISSUE TITLE
Broken typescript examples?

### ISSUE THREAD

**Issue by lamontadams** [ghengeveld/react-async#54]
Hi there.

Looking at the [example typescript app](https://github.com/ghengeveld/react-async/blob/master/examples/with-typescript/src/App.tsx). I'm getting compilation errors against typescript 3.4.3. It seems to be expecting `Async` to return a JSX element and it isn't, and there doesn't seem to be an `Async.Pending` property....

```
JSX element type 'Async<string>' is not a constructor function for JSX elements.
  Type 'Async<string>' is missing the following properties from type 'Element': type, key
JSX element type 'Async<string>' is not a constructor function for JSX elements.
Property 'Pending' does not exist on type '<T>(defaultProps?: AsyncProps<T> | undefined) => Async<T>'.
Property 'Pending' does not exist on type '<T>(defaultProps?: AsyncProps<T> | undefined) => Async<T>'.
```
I'm looking at tsconfig.json and I'm not seeing anything there that should make a difference (this is also a create-react-app app). Should that example work as-is?

**Comment by lamontadams** [comment]
Didn't notice 3.5 had dropped when I posted this. I'm getting the same errors with tsc version 3.5.1

**Comment by dobesv** [comment]
Currently we cannot use the `Async` component or the `createInstance` function since their typescript declarations are incorrect.

**Comment by ghengeveld** [comment]
This should be fixed in [v6.2.1](https://github.com/ghengeveld/react-async/releases/tag/v6.2.1).

Will you please verify?

**Comment by ghengeveld** [comment]
Turned out there was another issue with element types, which is fixed in [v6.2.2](https://github.com/ghengeveld/react-async/releases/tag/v6.2.2).
